addon/templates/components/frost-select.hbs

```diff
@@ -22,15 +22,15 @@
     <div class="up-arrow"></div>
     <ul ref="container" style={{containerStyle}}>
       {{#each displayItems as |item|}}
-        <li
-          class={{item.className}}
-          data-value={{item.value}}
-          data-index={{item.index}}
-          onClick={{action 'onSelect'}}
-          onMouseOver={{action 'onItemOver'}}
-        >
+        {{#frost-select-li
```

Comment:
Before merge to core I'd like to have this renamed to `frost-select-item` instead of `li`